app/src/main/java/org/brightmindenrichment/MigrationUtils.kt

```diff
@@ -0,0 +1,75 @@
+package org.brightmindenrichment.street_care.ui.visit
+
+import android.util.Log
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.QuerySnapshot
+
+
+/**
+ * Utility for migrating specific fields in Firestore documents.
+ *
+ * **Use with caution**: This can modify live data. Always test in a non-production environment.
+ *
+ * Example usage:
+ *
+ * ```
+ * MigrationUtils.migrateSpecificDataForAllUsers<String, String>({
+ *     when (it) {
+ *         "Counselling and Mentoring" -> "Counseling and Mentoring"
+ *         else -> it
+ *     }
+ * }, "helpType", "outreachEventsDev")
+ * ```
+ */
+object MigrationUtils {
+    val firestore: FirebaseFirestore = FirebaseFirestore.getInstance()
+    /**
+     * Generic migration function to update specific fields in Firestore documents.
+     *
+     * @param mapToCorrectValue A lambda function that takes a value of type T and returns a value of type R.
+     * @param fieldToUpdate The name of the field to update in the Firestore document.
+     * @param collection The name of the Firestore collection.
+     *
+     * @param T The original type of the field value.
+     * @param R The type of the transformed value after applying mapToCorrectValue.
+     */
+    fun <T, R> migrateSpecificDataForAllUsers(
```

Comment:
How would this be ran?

`
fun main() {
    MigrationUtils.migrateSpecificDataForAllUsers<String, String>(
        mapToCorrectValue = {
            when (it) {
                "food needed" -> "Food Donations"
                else -> it
            }
        },
        fieldToUpdate = "helpType",
        collection = "outreachEventsAndroid"
    )
}
`

Reply:
This can be called form MainActivity onCreate() fun!